requirements.txt

```diff
@@ -0,0 +1 @@
+pre-commit==0.13.3
```

Comment:
I'd probably call this file `requirements-dev.txt` since you don't actually need pre-commit to be installed in production (and that's what `requirements.txt` implies, even if you're not actually doing it in this case).